crypto/chacha20poly1305/chacha_merged.c

```diff
@@ -6,6 +6,7 @@ Public domain.
 
 #include "ecrypt-sync.h"
 #include "ecrypt-portable.h"
+#include <stddef.h>
```

Comment:
??

Reply:
When I compiled the firmware I got a `NULL` not defined exception. This is my way to resolve the error temporarily